newm-server/src/main/kotlin/io/newm/server/features/release/model/SpotifyAuthResponse.kt

```diff
@@ -0,0 +1,12 @@
+package io.newm.server.features.release.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SpotifyAuthResponse(
```

Comment:
See `TokenInfo`. I think some of this auth code is duplicated in SpotifyProfileUrlVerifier.